openbb_platform/providers/bis/pyproject.toml

```
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "openbb-bis"
+version = "1.0.0"
+description = "BIS Extension for the OpenBB Platform"
+authors = ["finestday <179567039+finestday@users.noreply.github.com>"]
+readme = "README.md"
+packages = [{ include = "openbb_bis" }]
+
+[tool.poetry.dependencies]
+python = "^3.8"
```

We currently support versions of Python between 3.10 and 3.13.
```suggestion
python = ">=3.10,<3.14"
openbb-core = "^1.5.2"
```